Cargo.lock

```diff
@@ -610,7 +610,7 @@ dependencies = [
 [[package]]
 name = "kvproto"
 version = "0.0.1"
-source = "git+https://github.com/pingcap/kvproto.git#0ba3ca8a6e3775accbe4bdf844162cd7685dafe2"
+source = "git+https://github.com/MyonKeminta/kvproto.git?branch=misono/add-end-key#1ac02db4d1907c5d6e2b7d09a678266cca2bc9f6"
```

Comment:
Use your own repository?

Reply:
This will be changed back to our repo after merging kvproto's PR. It's temporarily changed to my own branch to speedup reviewing. We always do this. Maybe I should tag a DNM